src/VoltServiceProvider.php

```diff
@@ -116,7 +116,7 @@ protected function registerTestingMacros(): void
                 $component = FragmentMap::get($component);
             }
 
-            return $this->assertSeeLivewire($component); // @phpstan-ignore-line
+            return $this->assertSeeLivewire($component);
```

Comment:
Why was this change made?

Reply:
The static analysis in the github action for the PR threw an error because there was nothing to ignore anymore. I suspect the original suppression was because of the use of final? Happy to revert